server/apps/frontend/src/Seller/SellerForm/SellerFrom.hook.ts

```diff
@@ -0,0 +1,63 @@
+import React, { FormEvent, RefObject } from 'react';
+import { useMutation } from 'react-query';
+
+import { AddSellerType, SellerForm } from '../Seller.hook';
+
+export const useForm = (
+  addSeller: AddSellerType
+): {
+  nameRef: RefObject<HTMLInputElement>;
+  passwordRef: RefObject<HTMLInputElement>;
+  urlRef: RefObject<HTMLInputElement>;
+  handleSubmit: (event: FormEvent) => void;
+  isError: boolean;
+} => {
+  const nameRef: RefObject<HTMLInputElement> = React.useRef(null);
+  const passwordRef: RefObject<HTMLInputElement> = React.useRef(null);
+  const urlRef: RefObject<HTMLInputElement> = React.useRef(null);
+  const [isError, setIsError] = React.useState<boolean>(false);
```

Comment:
It's not clear with the name what's in error. 

```suggestion
  const [hasError, setHasError] = React.useState<boolean>(false);
```